packages/state-react/src/lib/useAtom.ts

```diff
@@ -4,7 +4,7 @@ import { useState } from 'react'
 /**
  * Creates a new atom and returns it. The atom will be created only once.
  *
- * See {@link state#atom}
```

Comment:
You want `{@link @tldraw/state#atom}` I believe - probably better for this to link?